pages/glossary/default-element-language.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Default Element Language
+key: default-element-language
+unambiguous: true
+objective: true
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+---
+
+The _default language of an element_ E is the _most common language_ used in the texts of its _candidate elements_. If there are no _words_ in the candidate elements, the default language is undefined.
+
+The _most common language_ is determined by counting the number of _words_ in the candidate elements that are part of any of the languages in the [language subtag registry][]. The same word can be part of multiple languages. If the number of words is the same, the default language is undefined.
+
+The _candidates elements_ are all the elements for which all the following are true:
+
+- the element is an [inclusive descendant][] of E in the [flat tree][]; and
+- the element has no [inclusive ancestor][] in the [flat tree][] which is a descendant of E and has a non-empty (`""`) `lang` attribute (that is, no `lang` attribute "between" them).
+
+The _words_ used in the following texts should be counted for the _most common language_:
+
+- **text nodes**: the value of any [text nodes][] that are [visible][] or [included in the accessibility tree][] and children of a candidate element;
+- **accessible text**: the [accessible name][] and [accessible description][] of any candidate element [included in the accessibility tree][];
+- **page title**: the value of the [document title][], only if E is a [document][] in a [top-level browsing context][].
```

Comment:
Why do we want to count the page title for determining the language of an element that is not the document itself?
This can lead to wrong language identification when a page as a long title and I'm just marking the language of a small passage.

Reply:
That's he other way around. We count the page title **only** for determining the language of a document. Not for other elements.

Reply:
You are correct. I must have been too tired already :-)